Error: When your component includes a collection of radio `input` elements with the same name, the "radiogroup" `role` is unnecessary. You only need it if you are not using true form controls. The "group" `role` is sufficient.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
diff --git a/output/Radio Group/4/618bcb9b-4963-4000-8d5f-973641a35113.html b/output/Radio Group/4/618bcb9b-4963-4000-8d5f-973641a35113.html
index 5cfd106..4fef8ae 100644
--- a/output/Radio Group/4/618bcb9b-4963-4000-8d5f-973641a35113.html	
+++ b/output/Radio Group/4/618bcb9b-4963-4000-8d5f-973641a35113.html	
@@ -1,15 +1,15 @@
-<div role="radiogroup" aria-labelledby="theme-label">
+<div role="group" aria-labelledby="theme-label">
   <div id="theme-label">Choose a theme:</div>
-  <label>
-    <input type="radio" name="theme" value="light">
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light">
     Light
   </label>
-  <label>
-    <input type="radio" name="theme" value="dark">
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark">
     Dark
   </label>
-  <label>
-    <input type="radio" name="theme" value="high-contrast">
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast">
     High Contrast
   </label>
 </div>
\ No newline at end of file
